plugins/catalog-graph/src/components/EntityRelationsGraph/useEntityRelationGraph.ts

```diff
@@ -24,13 +25,14 @@ import { useEntityStore } from './useEntityStore';
  */
 export function useEntityRelationGraph({
   rootEntityRefs,
-  filter: { maxDepth = Number.POSITIVE_INFINITY, relations, kinds } = {},
+  filter: { maxDepth = Number.POSITIVE_INFINITY, relations, kinds, func } = {},
 }: {
   rootEntityRefs: string[];
   filter?: {
     maxDepth?: number;
     relations?: string[];
     kinds?: string[];
+    func?: (entity: Entity) => boolean;
```

Comment:
Better name to use than `func` here?

Reply:
would use the same name as in component `filterFunc` or what ever it's renamed to.